lrpar/doc/src/quickstart.md

```diff
@@ -0,0 +1,94 @@
+# Quickstart Guide
+
+This is a short guide to get you started. For a more detailed explanation about
+what's going on, please refer to the [tutorial](tutorial/index.html).
+
+We begin by creating a `Cargo.toml` file and load the necessary dependencies:
+
+```toml
+[package]
+name = "project"
+version = "0.0.1"
+authors = ["<authors>"]
+
+[[bin]]
+doc = false
+name = "project"
+
+[build-dependencies]
+lrlex = { path="<path to lrlex>" }
+lrpar = { path="<path to lrpar>" }
+
+[dependencies]
+cfgrammar = { path="<path to cfgrammar>" }
+lrlex = { path="<path to lrlex>" }
+lrpar = { path="<path to lrpar>" }
+```
+
+In the root of the project we create a file `build.rs`, which processes our lexer and grammar files.
+Here we can also choose the integer size for the token ID's and whether we want to generate a default
+parse tree, or use grammar actions instead:
+
+```rust
+extern crate lrlex;
+extern crate lrpar;
+
+use lrlex::LexerBuilder;
+use lrpar::{CTParserBuilder, ActionKind};
+
+fn main() -> Result<(), Box<std::error::Error>> {
+    // We specify the integer type for token ID's as u8, which suffices for our
+    // purposes. Note, that this type *must* be big enough to fit all token
+    // IDs in.
+    let lex_rule_ids_map = CTParserBuilder::<u8>::new().action_kind(ActionKind::CustomAction).process_file_in_src("project.y")?;
```

Comment:
If you're using `CustomAction`, it doesn't matter too much (I think) what integer size you use, because you're not storing many of them a go, so I would drop this entirely (including the sentence earlier). It will default to `u32` if not specified.